android/build.gradle

```diff
@@ -34,6 +34,12 @@ repositories {
 dependencies {
     implementation 'com.braintreepayments.api:drop-in:4.+'
     implementation 'com.facebook.react:react-native:+'
+    // Braintree dependencies
+    implementation 'com.braintreepayments.api:braintree:3.7.0'
+    implementation 'com.braintreepayments.api:google-payment:3.2.0'
+
+    // Google dependencies
+    implementation 'com.google.android.gms:play-services-wallet:16.0.1'
```

Comment:
Please don't force the version

See [example](https://github.com/react-native-community/react-native-camera/blob/master/android/build.gradle#L73)